### PR TITLE
Added inputAndroidContainer for renderAndroidHeadless()

### LINKED ISSUE #49: renderAndroidHeadless() does not allow you to give styles to the container.
The renderAndroidHeadless() returns a View that cannot be customised while the renderTextInputOrChildren() used in renderIOS() allows you to change the style of the view using this.props.style.inputIOSContainer. It would be nice to also customise the android view. 

 In renderAndroidHeadless():
```
 return (
            <View style={{ borderWidth: 0 }}>
                {this.props.children}
                ...
            </View>
        );
```

In renderTextInputOrChildren():
```
return (
                <View pointerEvents="box-only" style={this.props.style.inputIOSContainer}>
                    {this.props.children}
                </View>
            );
```

### PR DESCRIPTION
**Pull request by craig95**
Fixes #49

### CODE REVIEW

**Comment by lfkwtz** [comment]
Want to include #48 in this PR?

File: src/index.js [hunk 1]
```diff
@@ -271,7 +271,7 @@ export default class RNPickerSelect extends PureComponent {
 
     renderAndroidHeadless() {
         return (
-            <View style={{ borderWidth: 0 }}>
+            <View style={[{ borderWidth: 0 }, this.props.style.inputAndroidContainer]}>
```
Comment (lfkwtz) [inline src/index.js]: call this `headlessAndroidContainer` instead

Comment (craig95) [reply src/index.js]: 👍

**Comment by craig95** [comment]
Added #48

**Comment by lfkwtz** [comment]
Looks good. Just remove those extraneous formatting changes.

**Comment by craig95** [comment]
Not sure what you mean by "extraneous formatting changes".

**Comment by lfkwtz** [comment]
All good. And remove package.json from the Pr. Looks like there aren’t any changes there.

**Comment by craig95** [comment]
Removed the changes from the package.json from the Pr. 👍

**Pull request merged**